core/lexicon/en/setting.inc.php

```diff
@@ -381,8 +381,8 @@
 $_lang['setting_mail_smtp_port'] = 'SMTP Port';
 $_lang['setting_mail_smtp_port_desc'] = 'Sets the default SMTP server port.';
 
-$_lang['setting_mail_smtp_prefix'] = 'SMTP Connection Prefix';
-$_lang['setting_mail_smtp_prefix_desc'] = 'Sets connection prefix. Options are "", "ssl" or "tls"';
+$_lang['setting_mail_smtp_prefix'] = 'SMTP Encryption';
```

Comment:
I would suggest keeping the old name for a logical connection to the setting name.

Reply:
The setting key should be changed too. The setting that is changed with this is SMTPSecure in PHPMailer 6.

Reply:
> The setting key should be changed too. The setting that is changed with this is SMTPSecure in PHPMailer 6.

Are you talking about the "setting_mail_smtp_prefix" key or the "SMTP Connection Prefix" name?

Reply:
This MODX system setting changes value for the PHPMailer 6 property named SMTPSecure. The text of the MODX setting is misleading. Maybe 'SMTP Connection Prefix' was the right text somewhen in the past. Now the PHPMailer property changes the encryption of the SMTP connection.

Reply:
I lean towards updating the Lexicon key also, and would suggest `setting_mail_smtp_secure` to match PHP Mailer. This will lead to a little less confusing for future contributors. The language of the setting is clear to me.

This (obviously) would require changing the key in the the relevant Manager page, also.

Reply:
I concur with changing the key of the lexicon and altering the setting key itself from mail_smtp_prefix to mail_smtp_secure.